server/src/main/java/org/opensearch/bootstrap/Bootstrap.java

```diff
@@ -263,7 +263,7 @@ static SecureSettings loadSecureSettings(Environment initialEnv) throws Bootstra
         SecureString password;
         try {
             if (keystore != null && keystore.hasPassword()) {
-                password = readPassphrase(System.in, KeyStoreAwareCommand.MAX_PASSPHRASE_LENGTH);
+                password = readPassphrase(System.in, KeystoreConstants.MAX_PASSPHRASE_LENGTH);
```

Comment:
This is the only place `MAX_PASSPHRASE_LENGTH` is used, right? If yes, let's move it here

Reply:
It's also used in the `KeyStoreAwareCommand#readPassword` method:

```
            passwordArray = terminal.readSecret(
                "Enter new password for the opensearch keystore (empty for no password): ",
                MAX_PASSPHRASE_LENGTH
            );
            char[] passwordVerification = terminal.readSecret("Enter same password again: ", MAX_PASSPHRASE_LENGTH);
```